test.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/alphagov/verify/ruby:2.6.6
+FROM ruby:2.6.6
```

Comment:
This too: https://github.com/alphagov/verify-terraform/blob/master/environments/tools/platform/files/pipelines/verify-self-service.yml#L140